### ISSUE TITLE
Can't save files when extension is enabled

### ISSUE THREAD

**Issue by jpap** [beerose/go-group-imports-vscode#8]
I am using the VSCode version below.  When this extension is enabled, saving a file just doesn't work.  I didn't see anything related to this in the Output panel for the Window or otherwise.

I worked out that the save-fail cause was this extension by disabling all of my extensions, then re-enabling them one-by-one, until I converged on this one.  I am using GOPATH.

I'm happy to do anything to test/help.  I love your work and would love to see go-group-imports working again!

```
Version: 1.44.0
Commit: 2aae1f26c72891c399f860409176fe435a154b13
Date: 2020-04-08T08:23:56.137Z
Electron: 7.1.11
Chrome: 78.0.3904.130
Node.js: 12.8.1
V8: 7.8.279.23-electron.0
OS: Darwin x64 19.4.0
```

**Comment by beerose** [comment]
Hi @jpap,
I was able to reproduce this. The case is that this extension conflicts with some extensions that are adding a new line to the file. Will work on the fix. 
Thanks for submitting the issue!

**Comment by beerose** [comment]
@jpap 
I just fixed this and published `go-group-imports@1.1.3`. Feel free to reopen if it happens again.

**Comment by jpap** [comment]
Thanks for your super-fast work on this -- the new version works perfectly, thank you! :)
